composer/trainer/trainer.py

```diff
@@ -948,20 +962,35 @@ def __init__(
 
         # Console Logging
         loggers = list(ensure_tuple(loggers))
+
+        if progress_bar and log_to_console:
+            warnings.warn(
+                'Setting both `progress_bar` and `log_to_console` both to True is not recommended and will'
+                'lead to duplicate logs and weird formatting issues. Please set one of them to False for a better logging experience.'
+            )
+
         if any(isinstance(x, ProgressBarLogger) for x in loggers):
             warnings.warn(
                 DeprecationWarning(
                     (f'Specifying the {ProgressBarLogger.__name__} via `loggers` is deprecated. Instead, '
-                     'please specify `progress_bar`, `log_to_console`, and `stream` arguments when '
+                     'please specify `progress_bar` and `stream` arguments when '
                      'constructing the trainer. If specified, these arguments will be ignored, as the '
                      f'{ProgressBarLogger.__name__} was already created.')))
         else:
-            loggers.append(
-                ProgressBarLogger(
-                    progress_bar=progress_bar,
-                    log_to_console=log_to_console,
-                    stream=console_stream,
-                ))
+            if progress_bar:
+                loggers.append(ProgressBarLogger(stream=console_stream, log_traces=log_traces))
+
+        # Console Logging
+        if any(isinstance(x, ConsoleLogger) for x in loggers):
+            warnings.warn(
+                DeprecationWarning((f'Specifying the {ConsoleLogger.__name__} via `loggers` is deprecated. Instead, '
+                                    'please specify `log_to_console`, and `stream` arguments when '
```

Comment:
the arguments are `log_to_console`, `console_stream`, `console_log_interval`, and `log_traces`, right?